src/app/components/users/views/login.jade

```diff
@@ -0,0 +1,44 @@
+div.container
+  div.panel.panel-default
+    div.panel-heading
+      h2.text-center
+        | {{ 'LOGIN_HEADER' | translate }}
+      form.panel-body(name="loginForm"
+                      ng-submit="linCtrl.submitForm(loginForm.$valid)"
+                      novalidate)
+
+        div.alert-danger.text-center(ng-show="linCtrl.message")
+          | {{ linCtrl.message }}
+
+        div.form-group(ng-class="{ 'has-error': loginForm.email.$invalid && loginForm.email.$touched }")
+          label(for="inputEmail")
+            | Email
+          input.form-control(type="email"
+                             name="email"
+                             placeholder="Email"
+                             id="inputEmail"
+                             ng-model="linCtrl.form.email"
+                             required)
+          div.help-block(ng-messages="loginForm.email.$error"
+                         ng-if="loginForm.email.$touched")
+            p(ng-message="required")
+              | {{ 'REQUIRED_EMAIL_MSG' | translate }}
```

Comment:
when you have just translated text inside a tag you can put it this way
`p(ng-message="required" translate="REQUIRED_EMAIL_MSG")`
It's the prefered way